packages/appkit/src/client/appkit-base-client.ts

```diff
@@ -867,7 +873,10 @@ export abstract class AppKitBaseClient {
     }
 
     const connectionStatus = StorageUtil.getConnectionStatus()
-    if (connectionStatus === 'connected') {
+
+    if (!this.options?.enableReconnect) {
```

Comment:
shouldn't this be `this.options?.enableReconnect === false` ?

Reply:
`enableReconnect` is enabled by default now. So actually I should remove `!` here

Reply:
Hmm yeah I got you right